akka-stream/src/main/scala/akka/stream/impl/FanoutProcessor.scala

```diff
@@ -118,13 +119,14 @@ import org.reactivestreams.Subscriber
 /**
  * INTERNAL API
  */
-@InternalApi private[akka] class FanoutProcessorImpl(attributes: Attributes) extends ActorProcessorImpl(attributes) {
+@InternalApi private[akka] class FanoutProcessorImpl(attributes: Attributes)
+    extends ActorProcessorImpl(attributes)
+    with ActorTimers {
```

Comment:
adds a bit of weight and `aroundReceive` to the FanoutProcessorImpl, but that should be ok

alternative would be to keep track of the Cancellable, and cancel it in postStop